### DISCUSSION TITLE
OAuth must be provided error

### DISCUSSION THREAD

**Discussion by polklabs** [Taxel/PlexTraktSync#1327] [Support] [answered]
When trying to run sync command in docker i get the following error.
`trakt.errors.OAuthException: Unauthorized - OAuth must be provided`

Not sure if this is due to Trakt being down recently. I've tried running the login command which just returns:
`Checking Plex and Trakt logins

Success!
`

Is there a way for me to reset my Trakt credentials? I tried completely removing the image and config but it still thinks I'm logged in

**Comment by kushfest** [comment]
I'm getting the same error and am also seeing "Success!" when I run plextraktsync login. I tried removing my file .pytrakt.json and re-authenticated trakt, but I'm getting the same error. Nothing appears to scrobble, even though I see the "on_play" activity in my plextraktsync logs. Any suggestions?

**Reply by kushfest** [reply]
Interestingly, this seems to have resolved itself.

**Comment by galacticat** [comment, marked as answer]
I think I figured this out.

1. navigate to your config directory 
2. rename the .env and the .pytrakt.json files
3. Run plextraktsync with the login option again
4. I don't know if it is "required" but the new prompt instructions says to call the Trakt app "plex"

**Reply by polklabs** [reply]
This was it.  .pytrakt.json is a hidden file in windows so I didn't know it was there when trying to clear the config files

**Reply by pax0707** [reply]
Does not need to be recreated. Just enter info from the existing app.
